zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/listeners/task/TaskListenerTest.java

```diff
@@ -163,6 +163,8 @@ public void shouldAssignUserTaskAfterAllAssignmentTaskListenersAreExecuted() {
             Assertions.assertThat(userTask)
                 .hasAssignee("me")
                 .hasAction("my_assign_action")
+                .describedAs(
+                    "Expected all variables provided during UserTask assignment to be ignored")
```

Comment:
💭 Yeah, variables aren't support for the assign command. However, it's not exactly related to task listeners but to task assignment in general. So, I'd be fine with removing the variables from this test completely.